src/commands/app/deploy.js

```diff
@@ -114,6 +115,20 @@ class Deploy extends BuildCommand {
       } else {
         this.log('skipping publish phase...')
       }
+
+      // send audit log event for successful deploy
+      try {
+        const cliDetails = await getCliInfo()
```

Comment:
This should be done after getAuditLogEvent, because if logEvent is null, we can't do anything with it.

Reply:
also, getCliInfo could potential force a login, which may not be what the user (or CI/CD) wants.  If deploy is called with --no-publish then a login is not required (just RT credentials)